docs/compute/persistent_tpm_and_uefi_state.md

```diff
@@ -45,6 +45,18 @@ spec:
             persistent: true
 ```
 
+
+In order for the persistent tpm volume to be created successfully you must ensure your storage classes and [storage profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) are configured correctly.  
+The persistent tpm volume will be created with the below access mode if one of the constraints for the access mode is true.
+
+* RWX (ReadWriteMany):
+    * the respective storage profile has any claim property set with Filesystem volume mode and RWX access mode.
+    * the kubevirt cluster config has `VMStateStorageClass` set and the storage profile does not exist or the storage profile exists but has no claim property sets.
+
+* RWO (ReadWriteOnce):
+    * the respective storage profile has claim property sets with Filesystem volume mode and RWO access mode but not RWX.
+    * the kubevirt cluster config has `VMStateStorageClass` **unset** and the storage profile does not exist or the storage profile exists but has no claim property sets.
```

Comment:
nit - set

Reply:
I agree it's debatable above, but less so here. An empty claimPropertySets section has no claim property set... Maybe reword to avoid the confusion, like "has no claim property set defined" or "has an empty claimPropertySets list"?